src/openhmdi.h

```diff
@@ -143,6 +143,7 @@ void ohmd_set_default_device_properties(ohmd_device_properties* props);
 void ohmd_calc_default_proj_matrices(ohmd_device_properties* props);
 void ohmd_set_universal_distortion_k(ohmd_device_properties* props, float a, float b, float c, float d);
 void ohmd_set_universal_aberration_k(ohmd_device_properties* props, float r, float g, float b);
+int ohmd_wstring_match(const wchar_t *a, const wchar_t *b);
```

Comment:
This should be implemented in a new utils.h instead of in openhmdi.h (which is just for external facing API).
I was planning in general to move much shared functions to a new utils file, so this would be a good start.

Reply:
You might be thinking of include/openhmd.h? openhmdi.h isn't exported API

Reply:
What's the consensus on how to proceed?

Reply:
I'm good with it how it is, so it's waiting on @TheOnlyJoey

Reply:
Hmm since the openhmdi.h is a template for the internal interface i think it is more clear to implement this into a new utils.h to keep it seperated. (for multiplatform functions, platform specific should still be implemented in the respectful platform-* files.)